src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component } from "react";
+
+interface ErrorBoundaryProps {
+  children: any;
+}
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  constructor(props: ErrorBoundaryProps) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return <h1>Something went wrong.</h1>;
+    }
```

Comment:
I did could not make this message rendered, please explain how to reproduce the test scenario

Reply:
I don't know I am doing right, but I just tried to throw an error.
![image](https://user-images.githubusercontent.com/115835493/200665169-b5a1bfe1-11f2-4bb3-afc2-39b140e6a4f5.png)

Reply:
ah that is correct!